set.mm

```diff
@@ -89475,15 +89486,28 @@ being the empty set ( ` A e/ _V ` ).  (Contributed by AV, 6-Aug-2019.) $)
   ${
     $d A x $.  $d B x $.
     $( A subset of a natural number is finite.  (Contributed by NM,
-       24-Jun-1998.) $)
+       24-Jun-1998.)  (Revised by BTernaryTau, 23-Sep-2024.) $)
```

Comment:
Tag should be "(Proof shortened by BTernaryTau, 23-Sep-2024.)" instead of "(Revised by BTernaryTau, 23-Sep-2024.)"